programs/survival/instamine_deepslate.sc

```diff
@@ -0,0 +1,58 @@
+// allows instamining deepslate if the player has an efficiency 5 netherite pick and haste 2
+// works even if the player doesn't have carpet mod on their client
+
+__config() -> {
+  'stay_loaded' -> true,
```

Comment:
Default value is true

Reply:
This script doesn't load automatically without this

Reply:
That shouldn't be the case, it's true by default since ages ago.

Reply:
Resolved in Discord (it happens because of a carpet bug where it defaults to false with no `__config`).